ygnmi/ygnmi.go

```diff
@@ -33,8 +33,11 @@ import (
 // Supported operations: Batch.
 type AnyQuery[T any] interface {
 	// pathStruct returns to path struct for this query.
```

Comment:
returns the path struct corresponding to the return type for this query.

Reply:
True, I was thinking about the return type for Lookup/Watch etc. functions. Perhaps "corresponding to T"?

Reply:
done